modin/pandas/test/test_dataframe.py

```diff
@@ -1440,6 +1440,25 @@ def test_insert(self, data, loc):
         modin_df = pd.DataFrame(data)
         pandas_df = pandas.DataFrame(data)
 
+        astypes = ["category", "int32", "float"]
```

Comment:
The more common approach to this style of test is `pytest.mark.parametrize`: https://docs.pytest.org/en/latest/parametrize.html

Reply:
Yes, but notice that `astypes` affects only on a small part of `test_insert`, if we move that to `parametrize` we would run the same tests for `len(astypes)` times.
Is this applicable approach, or should we move the part that tests inserting of different `dtype` to a separate test function, for example `test_insert_dtype_save`?

What do you think?